coalahtml/tree.py

```diff
@@ -0,0 +1,69 @@
+from collections import OrderedDict
+from queue import Queue
+import os
+
+
+class Node:
+
+    def __init__(self, name):
```

Comment:
If you want you can add parent and abspath as optional parameters. It may help you to use the node.